### PR TITLE
Rails 5 2 update

### PR DESCRIPTION
**Pull request by mdye**
Rails gem updated to 5.2 and necessary updates to changed files have been made, mostly impacting config/environment/*.rb configurations.

### CODE REVIEW

File: config/cable.yml
Comment (MisterMartin) [inline config/cable.yml:10]: Why "chord_testbed_production"? It is not a testbed anymore. I'd suggest a channel name of "earthcube_chords".

File: chords_start.sh
Comment (MisterMartin) [inline chords_start.sh:148]: Why is the database chords_ts_test? Shouldn't it be chords_ts_development?

Actually, why is this needed at all? Won't $influxdb_dbname be set to the correct database?

Comment (mdye) [reply chords_start.sh:148]: The reasoning behind this is the assumption that when CHORDS is started with the RAILS_ENV=development that development work is being performed and that the test suite will also be run. 

The way I was running the test suite was with the command:
docker-compose run -e "RAILS_ENV=test" app bundle exec rspec

This command doesn't ever execute the chords_start.sh script, and hence the chords_ts_test database is never created, and rspec exits with a failure to connect.

Creating the test database when the containers are brought up with RAILS_ENV=development may not be the proper way to do this, but it works. I'm totally open to doing this another way if there are other suggestions. (But I'm not sure it's worth the time to go down this rabbit hole at this point)

File: config/cable.yml
Comment (mdye) [reply config/cable.yml:10]: The cable.yml file was auto-generated by the rails 5.2 upgrate script, and to be honest, I'm not sure why it chose "testbed" as part of the prefix.

 I'm totally fine with using "earthcube_chords_production" for this value.

As a side note, there are a handful of other files that reference "testbed" that we may want to update.

File: chords_start.sh
Comment (MisterMartin) [reply chords_start.sh:148]: We should take a look at the various RAILS workflows (production, development and test). Perhaps the most consistent way to handle this is to make chords_control do the right thing for all of these. For now, this shouldn't hold up the merge of of this'll request.

File: config/cable.yml
Comment (MisterMartin) [reply config/cable.yml:10]: Aha, I see the other uses of testbed scattered round CHORDS. Most glaring is application module name in _application.rb_: `module ChordTestbed`. We will leave it as is for this PR. I'll open an issue for this.

**Review by MisterMartin — APPROVED** [review]
Reviewed.

**Pull request merged**
